src/Fixer/ControlStructure/NoBreakCommentFixer.php

```diff
@@ -169,7 +169,7 @@ private function isNoBreakComment(Token $token)
 
         $text = $this->configuration['comment_text'];
 
-        return preg_match("~^((//|#)\s*$text\s*)|(/\*\*?\s*$text\s*\*/)$~i", $token->getContent());
+        return preg_match("~^((//|#)\\s*$text\\s*)|(/\\*\\*?\\s*$text\\s*\\*/)$~i", $token->getContent());
```

Comment:
shouldn't we use `'` for string notation here?

Reply:
The string contains variables, I would keep using `"`.

Reply:
I think it would be better to write it like:
```php
sprintf('/my-regex-%s/', preg_quote($text));
```
I'm not a regex ninja, but I always had trobules on legacy code that doesn't escape regex variables.

Reply:
Nice catch! `preg_quote` added in #3263, thanks!